gm4_liquid_tanks/data/liquid_tanks/functions/create.mcfunction

```diff
@@ -0,0 +1,9 @@
+#@s = glass item inside correct hopper at armor stand location
+#run from main
+
+data merge block ~ ~ ~ {CustomName:'{"translate":"%1$s","with":["Empty Tank",{"translate":"block.gm4.liquid_tank.empty"}]}',Items:[]}
+summon armor_stand ~ ~ ~ {CustomName:"\"gm4_liquid_tank\"",Tags:["gm4_no_edit","gm4_liquid_tank","gm4_lt_idle"],NoGravity:1,Marker:1,Invisible:1,Invulnerable:1,Small:1,DisabledSlots:2039552,ArmorItems:[{},{},{},{id:"glass",Count:1}]}
```

Comment:
This glass block should have a custom model data applied to it. It can use id `1`.